plugins/pixart-rf/meson.build

```diff
@@ -1,8 +1,4 @@
 if get_option('plugin_pixart_rf') and host_machine.system() == 'linux'
-if not get_option('gudev')
-  error('gudev is required for plugin_pixart_rf')
-endif
```

Comment:
ohh, so we compile it even without gudev. I guess that makes sense.

Reply:
yeah when we go up to meson 0.59.0, there is some fancy stuff to allow us to drop this and auto-disable the plugin, but this is the best for now